homeassistant/components/wemo/fan.py

```diff
@@ -304,6 +304,8 @@ def turn_on(self, speed: str = None, **kwargs) -> None:
         else:
             self.set_speed(speed)
 
+        self.schedule_update_ha_state(True)
```

Comment:
Why are we calling this with `True`? This is exactly what will happen at the end of the service call by default.

The only change that could make sense is to update state without asking wemo for its state since that takes time and could take longer than the two second timeout that the frontend has for receiving the new state.